lightning/src/ln/channel.rs

```diff
@@ -4130,6 +4131,11 @@ impl<Signer: Sign> Channel<Signer> {
 		}
 	}
 
+	/// Get forwarding information for the counterparty.
+	pub fn get_counterparty_forwarding_info(&self) -> Option<CounterpartyForwardingInfo> {
```

Comment:
Not sure if we've been entirely consistent here, but consider dropping the `get_` prefix for accessor methods since that tends to be the Rust convention.